contracts/Automate.sol

```diff
@@ -166,6 +167,43 @@ contract Automate is Gelatofied, Proxied, AutomateStorage, IAutomate {
         );
     }
 
+    function exec1BalanceSimple(
+        address _taskCreator,
+        address _execAddress,
+        bytes memory _execData,
+        bytes32 _taskId,
+        Gelato1BalanceParam calldata _oneBalanceParam,
+        bool _revertOnFailure,
+        bool _singleExec
+    ) external onlyGelato {
+        require(
+            _createdTasks[_taskCreator].contains(_taskId),
+            "Automate.exec: Task not found"
+        );
+
+        bool success = LibSimpleTaskModule.onExecTask(
+            _taskId,
+            _taskCreator,
+            _execAddress,
+            _execData,
+            _revertOnFailure,
+            _singleExec,
+            taskModuleAddresses
+        );
+
+        emit LibEvents.ExecSuccess1Balance(_taskId, success);
+
+        emit LogUseGelato1Balance(
+            _oneBalanceParam.sponsor,
+            _execAddress,
+            _oneBalanceParam.feeToken,
+            _oneBalanceParam.oneBalanceChainId,
+            _oneBalanceParam.nativeToFeeTokenXRateNumerator,
+            _oneBalanceParam.nativeToFeeTokenXRateDenominator,
+            _oneBalanceParam.correlationId
+        );
```

Comment:
Maybe we could combine the 2 Events and only emit:
_taskId, success, correlationId

would need to check with Paul if `taskCreator` is necessary as well